pytest_splunk_addon/helmut/connector/sdk.py

```diff
@@ -94,27 +87,19 @@ def __init__(
         @param password: The password to use. If None (default)
                          L{Connector.DEFAULT_PASSWORD} is used.
         @type password: str
-        @param namespace: Deprecated. user owner and app instead.
-        @type namespace: str
         @param sharing: used by python sdk service
         @type sharing: str
         @param owner: used by python sdk service
         @type owner: str
         @param app: used by python sdk service
         @type app: str
         """
-
-        super(SDKConnector, self).__init__(
-            splunk, username=username, password=password, owner=owner, app=app
-        )
-        if namespace is not None and namespace != self.namespace:
-            msg = (
-                "namespace is derecated. please use owner and app. "
-                "Your namespace setting : %s, owner&app setting:%s"
-                % (namespace, self.namespace)
-            )
-            self.logger.error(msg)
-            raise Exception(msg)
+        self._splunk = splunk
+        self._username = username or self.DEFAULT_USERNAME
```

Comment:
why not default values like `sharing` arg?

Reply:
Done